components/NavBar/NavBar.js

```diff
@@ -31,15 +34,23 @@ export function NavBar() {
         </Link>
         <div>
           {searchActive ? (
-            <input
-              type="search"
-              onBlur={() => !hasValue && setSearchActive(false)}
-              placeholder="Search"
-              onChange={(e) => setInputValue(e.target.value)}
-              value={inputValue}
-              className={searchInput}
-              autoFocus
-            ></input>
+            <div className={inputSection}>
+              <input
+                type="text"
+                onBlur={() => !hasValue && setSearchActive(false)}
+                placeholder="Search"
+                onChange={(e) => setInputValue(e.target.value)}
+                value={inputValue}
+                className={searchInput}
+                autoFocus
+              ></input>
+              <div
+                className={cancelIconContainer}
+                onClick={() => setInputValue('')}
```

Comment:
It seems they do this too:

```suggestion
                onClick={() => {setInputValue(''); setSearchActive(false) }}
```